metron-analytics/metron-maas-service/README.md

```diff
@@ -138,7 +138,7 @@ Now that we have a deployed model, let's adjust the configurations for the Squid
 * Edit the squid parser configuration at `$METRON_HOME/config/zookeeper/parsers/squid.json` in your favorite text editor and add a new FieldTransformation to indicate a threat alert based on the model (note the addition of `is_malicious` and `is_alert`):
 ```
 {
-  "parserClassName": "org.apache.metron.parsers.GrokParser",
+  "parserClassName": "org.apache.metron.parsers.grok.GrokParser",
```

Comment:
As for a general rule -> Parsers the parser file formats, with no understanding of the content would be the rule.  "There could be any number of systems supported with this parser with that number of configurations".    Such parsers should also lend themselves to be extended in code.

- grok
- csv
- json
- xml
- scripting
- binary json
- protobuf